core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -293,16 +339,17 @@ public TenantObject setAdapterConfigurations(final List<Map<String, Object>> con
      *                       configured adapters or {@code null} in order to
      *                       remove any existing configuration.
      * @return This tenant for command chaining.
+     * @deprecated Use {@link #setAdapters(List adapters)} instead.
```

Comment:
IMHO this method now also may throw an `IllegalArgumentException` ...

Reply:
I have added info about this `IllegalArgumentException` to the javadoc of this method.